.github/workflows/odoo-ci-cd.yml

```
@@ -1,41 +1,318 @@
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
-
+        
       - name: Log in to GHCR
         uses: docker/login-action@v3
         with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+              registry: ghcr.io
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 14

(indentation)